### PR TITLE
docs: improve documentation for installation of unstable version

### PR DESCRIPTION
**Pull request by hugo-sid**

### CODE REVIEW

**Comment by hugo-sid** [comment]
Hi @jakubroztocil. This PR is ready for review.

**Comment by hugo-sid** [comment]
All checks have passed.

Could you please review this @jakubroztocil ?

Thanks.

File: docs/README.md [hunk 1]
```diff
@@ -250,36 +250,39 @@ $ pkg upgrade www/py-httpie
 
 ### Unstable version
 
-You can also install the latest unreleased development version directly from the `master` branch on GitHub.
-It is a work-in-progress of a future stable release so the experience might be not as smooth.
+If you want to try out the latest version of HTTPie that hasn't been officially released yet, you can install the development or unstable version directly from the master branch on GitHub. However, keep in mind that the development version is a work in progress and may not be as reliable as the stable version.
 
-You can install it on Linux, macOS, Windows, or FreeBSD with `pip`:
+You can use the following command to install the development version of HTTPie on Linux, macOS, Windows, or FreeBSD operating systems. With this command, the code present in the master branch is downloaded and installed using `pip`.
```
Comment (jakubroztocil) [inline docs/README.md:255]:
```suggestion
You can use the following command to install the development version of HTTPie on Linux, macOS, Windows, or FreeBSD operating systems. With this command, the code present in the `master` branch is downloaded and installed using `pip`.
```

**Comment by jakubroztocil** [comment]
Thanks, @hugo-sid!

**Pull request merged**